gemfiles/rails5.2.gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
 gem "activerecord", "~> 5.2.0", :require => "active_record"
-gem "sqlite3"
+gem "sqlite3", '~> 1.4'
```

Comment:
I usually use [the exact line](https://github.com/rails/rails/blob/5-2-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L12) from the corresponding version of sqlite3_adapter in Rails:

```
gem "sqlite3", "~> 1.3", ">= 1.3.6"
```